src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/PreviouslySoldOnArtsyRail.tsx

```diff
@@ -0,0 +1,215 @@
+                key={artwork.internalID}
+                artwork={artwork}
+                lazyLoad
+                data-testid="previouslySoldItem"
+                onClick={trackArtworkItemClick(artwork, i)}
+                // FIXME:
+                contextModule={ContextModule.artworkRecentlySoldGrid as any}
+              >
+                <Flex justifyContent="space-between" mb={[0.5, 1.5]}>
+                  <Text variant={["lg-display", "xl"]} overflowEllipsis>
+                    {priceRealized?.display}
+                  </Text>
+                  <Text
+                    variant={["lg-display", "xl"]}
+                    color="green100"
+                    overflowEllipsis
+                  >
+                    123%
+                  </Text>
+                </Flex>
+
+                <Text
+                  variant={["xs", "sm-display"]}
+                  color="black60"
+                  overflowEllipsis
+                  data-testid="previouslySoldItem-estimate"
+                >
+                  Estimate {lowEstimate?.display} - {highEstimate?.display}
+                </Text>
+              </SoldArtworkFragmentContainer>
+            )
+          }
+        )
+      }}
+    />
+  )
+}
+
+export const PreviouslySoldOnArtsyRailFragmentContainer = createFragmentContainer(
+  PreviouslySoldOnArtsyRail,
+  {
+    recentlySoldArtworks: graphql`
+      fragment PreviouslySoldOnArtsyRail_recentlySoldArtworks on RecentlySoldArtworkTypeConnection {
+        edges {
+          node {
+            artwork {
+              ...SoldArtwork_artwork
+              slug
+              internalID
+            }
+            lowEstimate {
+              display
+            }
+            highEstimate {
+              display
+            }
+            priceRealized {
+              display
+            }
+          }
+        }
+      }
+    `,
+  }
+)
+
+const PLACEHOLDER = (
+  <Skeleton>
+    <Rail
+      title="Previously sold on Artsy"
+      showProgress={false}
+      getItems={() => {
+        return [...new Array(20)].map((_, i) => {
+          return (
+            <Box
+              display="flex"
+              flexDirection="column"
+              justifyContent="flex-end"
+              width="100%"
+            >
+              <SkeletonBox width={[305, 362, 420]} height={[380, 495]} />
+
+              <Flex justifyContent="space-between" mb={[0.5, 1.5]}>
+                <Text variant={["lg-display", "xl"]} overflowEllipsis>
+                  $100,000
+                </Text>
+                <Text
+                  variant={["lg-display", "xl"]}
+                  color="green100"
+                  overflowEllipsis
+                >
+                  123%
+                </Text>
+              </Flex>
+
+              <Text
+                variant={["xs", "sm-display"]}
+                color="black60"
+                overflowEllipsis
+              >
+                Estimate $100,000–100,000
```

Comment:
Do we want to show values as placeholder?

Reply:
same comment, maybe we can just have those as a gray/black box as in app

Reply:
Oki, I'll use grey boxes